gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/avatarpicker/AvatarPickerViewModel.kt

```diff
@@ -34,6 +34,10 @@ internal class AvatarPickerViewModel(
     private val avatarRepository: AvatarRepository,
     private val fileUtils: FileUtils,
 ) : ViewModel() {
+    private companion object {
+        const val AVATAR_SWITCH_DELAY = 800L
```

Comment:
With less than 800 ms, I still see frequent issues.

Reply:
Thanks for testing 👍 
cc: @etoledom